front/app/components/FormBuilder/components/FormBuilderSettings/PointSettings/index.tsx

```diff
@@ -71,35 +78,52 @@ const PointSettings = ({ mapConfigIdName, field }: Props) => {
   }, [localize, mapConfig]);
 
   const onConfigureMapClick = useCallback(() => {
-    // Create a new map config if we don't have one for this field
+    // Create a new map config by duplicating the project config if we don't have one for this field
     if (!mapConfigId) {
-      // Initial data is from existing project map config
-      const initialData = projectMapConfig?.data?.attributes
-        ? projectMapConfig.data.attributes
-        : {};
-
-      createProjectMapConfig(
-        {
-          ...initialData,
-        },
-        {
+      const duplicateAndOpenModal = (projectMapConfigId: string) => {
+        duplicateMapConfig(projectMapConfigId, {
           onSuccess: (data) => {
             // Set the form value to the map config ID
             setValue(mapConfigIdName, data.data.id);
             // Open the modal
             setShowModal(true);
           },
-        }
-      );
+        });
+      };
+
+      const projectMapConfigId = projectMapConfig?.data?.id;
+      if (projectMapConfigId) {
+        duplicateAndOpenModal(projectMapConfigId);
+      } else {
+        // Create a default project map config if it doesn't exist
+        const defaultLatLng = getCenter(undefined, appConfig?.data, undefined);
+        const defaultZoom = getZoomLevel(undefined, appConfig?.data, undefined);
+        createMapConfig(
+          {
+            center_geojson: {
+              type: 'Point',
+              coordinates: [defaultLatLng[1], defaultLatLng[0]],
+            },
+            zoom_level: defaultZoom.toString(),
+          },
+          {
+            onSuccess: (newProjectMapConfig) => {
+              duplicateAndOpenModal(newProjectMapConfig.data.id);
+            },
```

Comment:
@jamesspeake So in this `onSuccess`, the createMapConfig() on line 101 isn't creating a project level map config - It's just creating a new map config not related to anything right?

So I don't believe we need to also duplicate it once it's been created? But maybe I'm wrong? :thinking:

Reply:
Hmm, maybe you're right, what it is supposed to do is create the default project map config if it does not exist, then it will duplicate the project map config as the field map config, but actually it does not seem to be setting the project ID anywhere. Will look at this.

Reply:
Ok :) I think though that we don't need to create the default project map config here (if it doesn't exist). It should be enough just to create a new map config like we're doing now, and then assign that map config ID to the survey question right (and then open the modal)?

Reply:
This was a change I made based on your previous comment. But wierdly I'm not getting the missing project map config any more when I create a new project. @jinjagit have you changed this? I thought it would be wise to make sure the project had a default config here and then just duplicate it instead of creating it.

Reply:
Ahh wait I see - it was from my change last week see [this commit](https://github.com/CitizenLabDotCo/citizenlab/pull/7305/commits/2d9e201fcde16c3c1eb85a8190cc089a3c953fc4).

It made more sense to me to just create a new config for the specific question if there wasn't an existing project-level config to duplicate. I missed that the "onSuccess" was then copying the created config though. Sorry about that James - my bad. 

If you feel strongly it should work that way (create a project config first if it doesn't exist and then duplicate that config) you can change this back to what you had :)

Reply:
@amanda-anderson I've changed this so that it's not duplicating any more. Although looks like the backend may now returns the default app config regardless of whether a project config exists in the DB or not, so this may not ever be called. Anyway, think the PR is now ready to go.

Reply:
Sounds good! Thanks @jamesspeake :) Did you want me to do a final functional review or anything?

Reply:
If you have time - I've done one myself and all seems to be working but probably helpful to have someone else look at it one last time

Reply:
Taking a look now! :) I'll post here when I'm done :+1:

Reply:
@jamesspeake Yep everything seems to be working for me! :partying_face: